spec/system/placements/support/schools/placements/support_user_views_a_placement_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe "Placements / Support / Schools / Placement / Support User views a placement",
+               type: :system, service: :placements do
+  let(:school) { create(:placements_school, name: "School 1", phase: "Nursery") }
```

Comment:
Phase should be "Primary", "Secondary" or "All-through". EYFS isn't handled by our service, so this should be change to reflect one of the three options I've mentioned.

Reply:
We _do_ handle it though, we import all schools so I think this is still a valid test

Reply:
I will change it, I just want to point out that whilst we're not actively onboarding these schools we have them in our database

Reply:
Stranger things have happened :D